examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -669,6 +669,13 @@ def parse_args(input_args=None):
         ),
     )
 
+    parser.add_argument(
+        "--image_interpolation_mode",
+        type=str,
+        default="lanczos",
+        help="The image interpolation method to use for resizing images.",
```

Comment:
Do we wanna mention `lanczos` is reasonably better than the others? Also, maybe we should provide a list of `choices`?